app/controllers/api/base_controller.rb

```diff
@@ -68,6 +68,10 @@ def authorization
     Authorization.new(prx_auth_token) if prx_auth_token
   end
 
+  def authenticate_user!
```

Comment:
Makes sense.  Probably more sense than having this in the concern, since `current_user` is defined here.